src/lib/maxim.ts

```
@@ -422,6 +424,8 @@ export class Maxim {
 							model: deployedVersion!.config?.model,
 							provider: deployedVersion!.config?.provider,
 							tags: deployedVersion!.config?.tags,
+							run: (input: string, options?: { imageUrls?: ImageUrl[]; variables?: { [key: string]: string } }) =>
+								this.APIService.prompt.runPromptVersion(deployedVersion!.id, input, options),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)